scene/gui/color_picker.cpp

```diff
@@ -380,6 +380,9 @@ void ColorPicker::_value_changed(double) {
 		if (sliders[1]->get_value() > 0 || sliders[0]->get_value() != cached_hue) {
 			cached_hue = sliders[0]->get_value();
 		}
+		if (sliders[2]->get_value() > 0 || sliders[1]->get_value() != cached_sat) {
```

Comment:
We could move this code to `ColorMode` by adding a virtual callback and call it here. But that's for another PR.

Reply:
I could start working on this at another time... what would it entail?

Reply:
Something like `void _value_changed()` method in ColorMode. It would be called here and implemented in ColorModeHSV and ColorModeOKHSL. This would allow to move `cached_hue` and `cached_saturation` out of ColorPicker and possibly same with `h`, `s`, `v` and `last_color` values (although I didn't check where exactly they are used).